### PR TITLE
Add Doko Demo Issho ,Driver 76, 	Harukanaru Toki no Naka de 3 with Izayoiki Aizouban into ForceUMDReadSpeed

### LINKED ISSUE #6127: HARUKANARU TOKI NO NAKA DE 3 WITH IZAYOIKI AIZOUBAN missing BGM and black screen after title
If you want to play this game on PPSSPP,you must disable BGM in PSP first,then copy savedata to PPSSPP,the game will run well.

Debug log:(rename jpg to rar)
ppsspplog

savedata with BGM disable.(rename jpg to rar)
uljm05441

### LINKED ISSUE #12054: Driver 76 freezing emulation in alleyways
### What happens?

When playing Driver 76, there seem to be numerous spots (most notably in alleyways) where the game goes awry and freezes or crashes the emulator completely. Here I have focused on a single, quick test case.

In the alleyway I tested against, game crashes with a 100% repro rate, albeit with at least two distinct results as per the log file (here with all channels set to **Warn** verbosity):

1:
```
16:27:996 idle0        W[G3D]: gpu\gpucommon.cpp:2295 Unknown GE command : fd19262f 
16:27:996 idle0        E[G3D]: gpu\gpucommon.cpp:1891 Bad bounding box data: c40000
```

(in this case PPSSPP effectively stalled in `GPUCommon::FastRunLoop` because `downcount` was set to an absurdly high value).

2:
```
33:23:460 idle0        W[G3D]: d3d11\statemappingd3d11.cpp:217 Unsupported RGB mask: r=00 g=32 b=01
33:23:468 idle0        W[G3D]: gpu\gpucommon.cpp:2295 Unknown GE command : 39900100 
33:23:478 idle0        E[G3D]: gpu\gpucommon.cpp:2599 BlockTransfer: Bad source transfer address 00000000!
33:23:478 idle0        E[G3D]: gpu\gpucommon.cpp:2599 BlockTransfer: Bad source transfer address 00000000!
33:23:478 idle0        E[G3D]: gpu\gpucommon.cpp:2599 BlockTransfer: Bad source transfer address 00000000!
33:23:478 idle0        E[G3D]: gpu\gpucommon.cpp:2599 BlockTransfer: Bad source transfer address 00000000!
33:23:478 idle0        E[G3D]: gpu\gpucommon.cpp:2599 BlockTransfer: Bad source transfer address 00000000!
33:23:478 idle0        E[G3D]: gpu\gpucommon.cpp:2599 BlockTransfer: Bad source transfer address 00000000!
33:23:478 idle0        E[G3D]: gpu\gpucommon.cpp:2599 BlockTransfer: Bad source transfer address 00000000!
33:23:478 idle0        E[G3D]: gpu\gpucommon.cpp:2599 BlockTransfer: Bad source transfer address 00000000!
33:23:487 idle0        E[G3D]: gpu\gpucommon.cpp:1891 Bad bounding box data: 010a00
33:23:487 idle0        E[G3D]: gpu\gpucommon.cpp:1246 CALL: Stack full!
33:23:487 idle0        E[G3D]: gpu\gpucommon.cpp:1246 CALL: Stack full!
33:23:487 idle0        E[G3D]: gpu\gpucommon.cpp:1246 CALL: Stack full!
[...Stack full error repeating in loop...]
```

Together with the freeze I can observe artifacts, but I don't know if those are related or not. They are mentioned in #7026 and #11399, but neither issue mentions anything about freezes.

I was unable to test it on PSP, but testing on PS Vita and Adrenaline didn't reveal any of those issues at both 222MHz and 333MHz (which seemingly is more than what game requests) CPU frequency.

### Reproduction steps

Pre: Tested game: Driver 76 (ULES00740001)
Attached savegame allowing to get to the crashing point within seconds:
[ULES00740001.zip](https://github.com/hrydgard/ppsspp/files/3203787/ULES00740001.zip)

1. Load the game and the save.
2. Start the first available mission ("All the way up to 11"), select "Just Drive".
3. Upon gaining control, drive forwards and turn right into the first alleyway.
4. Proceed through the alleyway and turn left. You may ignore the star, no need to collect it.
5. Observe severe graphical artifacts on the sky and game freezing shortly after.
5a. If you didn't crash as soon as you reach the first row of boxes, stop and reverse. After max 2 attempts, game freezes.

image

As far as usability goes, PPSSPP is left in a spinlocked, but responsive state. UI window can be closed, but it doesn't terminate the process because main thread joins on Emu thread, which in both instances is spinlocked. This leaves an "invisible" process hogging an entire CPU core in the background, which may be counter-intuitive for less tech savvy users. I don't know if it is required to join the Emu thread - perhaps it is better to detach it and let it be terminated non-gracefully by the OS when the main thread exits?

### What hardware, operating system, and PPSSPP version?  On desktop, GPU matters for graphical issues.

Windows 10 64-bit 1803, running the very latest PPSSPP x64 compiled from master (at the time of creating this issue, f7a2fc963d6ddd514e5f9fec119c2a8120329ae2). Same issue happens on a public 1.8.0 build.

On top of that, I have tested all of the following:
- 32-bit and 64-bit PPSSPP builds.
- Vulkan and D3D11 backends.
- PSP-1000 and PSP-2000/3000 modes.
- AUTO and 222MHz CPU frequency (reportedly D76 freezes a lot on real hardware with CPU frequency forced to 333MHz, too).

None of the following resolved freezing.

### LINKED ISSUE #18420: Blue screen after first save (UCJS10002 : Doko Demo Issho)
### Game or games this happens in

UCJS10002 : どこでもいっしょ(Doko Demo Issho)

### What area of the game / PPSSPP

After I saved my game for the first time, I can't open the game without a savestate. It has a blue screen every time. (thats very bad since the games whole point is checking on toro everyday :()

Screen Shot 2023-11-13 at 2 12 16 PM

heres my savedata: 

[UCJS10002000.zip](https://github.com/hrydgard/ppsspp/files/13340817/UCJS10002000.zip)

### What should happen

It should just open to the screen that asks if you want to continue. I can get to that screen via savestate: 
Screen Shot 2023-11-13 at 2 19 42 PM
(thats how it looks) But the crash makes it so I can't progress in the game. this crash never happened until I actually saved the game, and there was a continue button. 

### Logs

sadly I can't figure out where my dumped logs are on mac :,(

### Platform

macOS

### Mobile phone model or graphics card

Intel(R) Iris(TM) Graphics 6100

### PPSSPP version affected

v1.16.6

### Last working version

v1.16.6

### Graphics backend (3D API)

Vulkan

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [X] Try without any cheats and without loading any save states.
- [X] Include logs or screenshots of issue.

### PR DESCRIPTION
**Pull request by sum2012**
Fix #18420 Fix #12054 Fix #6127

### CODE REVIEW

File: assets/compat.ini [hunk 1]
```diff
@@ -1655,3 +1655,14 @@ ULJM06024 = true
...
 NPEG00036 = true
+
+# Doko Demo Issho,load savedata error see #18420
+UCJS10002 = true
+UCJS18002 = true
+
+# Driver '76 ,see #16904 #12054
+ULUS-10235 = true
+ULES-00740 = true
+ULES-00740/P = true
```
Comment (hrydgard) [inline assets/compat.ini:1666]: This /P thingy won't actually do anything and the ID is the same as above, so might as well delete it.

Comment (hrydgard) [reply assets/compat.ini:1666]: Also, please rmeove the dashes from the Driver IDs.

Did you test it with this?

Comment (anr2me) [reply assets/compat.ini:1666]: Looks like redump also have the ID with /P not sure whether it's a mistake or not http://redump.org/disc/7485/

Comment (sum2012) [reply assets/compat.ini:1666]: I asked my friend about  Platinum version, he answered me that ppsspp do not show /p in the gameid

**Comment by sum2012** [comment]
@hrydgard Sorry, I did not test again when I do pull request

**Review by hrydgard — APPROVED** [review]

**Pull request merged**